frontend/packages/console-app/src/components/cloud-shell/CloudShellTabHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Flex, FlexItem, FlexModifiers } from '@patternfly/react-core';
+import './CloudShellTab.scss';
+
+const CloudShellTabHeader = () => {
+  return (
```

Comment:
can we avoid explicit return statement and  have type `React.FC`

Reply:
updated